scripts/package/postinstall.sh

```diff
@@ -30,6 +30,358 @@ install() {
     rm -rf /usr/share/observiq-otel-collector
 }
 
+install_service() {
+  if command -v systemctl > /dev/null 2>&1; then
+    install_systemd_service
+  else
+    install_sysconfig_service
+    install_initd_service
```

Comment:
TBD if this is correct. Currently, we deploy all three service files to all nix systems, and use the `service` command to maange sysconfig or init.d, depending on what the platform expects. I suspect this is okay, and follows that same pattern.